soh/src/overlays/actors/ovl_En_Syateki_Man/z_en_syateki_man.c

```diff
@@ -371,7 +373,11 @@ void EnSyatekiMan_EndGame(EnSyatekiMan* this, GlobalContext* globalCtx) {
                             this->getItemId = GI_RUPEE_PURPLE;
                         }
                     }
-                    func_8002F434(&this->actor, globalCtx, this->getItemId, 2000.0f, 1000.0f);
+                    if (!gSaveContext.n64ddFlag) {
```

Comment:
Logical Error: Can it get to this point without setting getItemEntry?

Reply:
Same suggestion as for fishing.

Reply:
Only potential hurdle is where to set getItemEntry for this one, but you can probably just do it in this actor's init function.